docker/transmart-api-server/docker-entrypoint.sh

```diff
@@ -52,6 +54,10 @@ keycloak:
 EndOfMessage
 sync
 
+[[ -e "${CERTS_PATH}" ]] && \
```

Comment:
maybe better `-f` (to make sure it's file, not dir)?

Reply:
Yes, I'll change it.